packages/core/src/tracing/index.js

```diff
@@ -41,8 +41,10 @@ let processIdentityProvider = null;
 const instrumentations = [
   './instrumentation/cloud/aws-sdk/v2/index',
   './instrumentation/cloud/aws-sdk/v3/index',
+
```

Comment:
suggestion: can we avoid the empty lines in line no: 44 and 47?

Reply:
👍